.github/workflows/zizmor.yml

```diff
@@ -0,0 +1,32 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
```

Comment:
Is this 🌈 emoji actually part of the name?

Reply:
No it was part of the example in the zizmore repo

Reply:
Then can we remove it? It's a little distracting to have emojis in config files.

Reply:
Done